src/lib/angles.js

```diff
@@ -10,25 +10,119 @@
 
 var PI = Math.PI;
 
-exports.deg2rad = function(deg) {
+function deg2rad(deg) {
     return deg / 180 * PI;
-};
+}
 
-exports.rad2deg = function(rad) {
+function rad2deg(rad) {
     return rad / PI * 180;
-};
+}
 
-exports.wrap360 = function(deg) {
+function wrap360(deg) {
     var out = deg % 360;
     return out < 0 ? out + 360 : out;
-};
+}
 
-exports.wrap180 = function(deg) {
+function wrap180(deg) {
     if(Math.abs(deg) > 180) deg -= Math.round(deg / 360) * 360;
     return deg;
-};
+}
 
-exports.isFullCircle = function(sector) {
+/* is sector a full circle?
+ * ... this comes up a lot in SVG path-drawing routines
+ *
+ * @param {2-item array} sector sector angles in *degrees*
+ * @return {boolean}
+ */
+function isFullCircle(sector) {
     var arc = Math.abs(sector[1] - sector[0]);
     return arc === 360;
+}
+
+/* angular delta between angle 'a' and 'b'
+ *
+ * solution taken from: https://stackoverflow.com/a/2007279
+ *
+ * @param {number} a : first angle in *radians*
+ * @param {number} b : second angle in *radians*
+ * @return {number} angular delta in *radians*
+ */
+function angleDelta(a, b) {
+    var d = b - a;
+    return Math.atan2(Math.sin(d), Math.cos(d));
```

Comment:
hmm OK... I'm sure it's not used enough to really matter but this feels heavy just to wrap to `[-π,π]` - can it just be `Lib.mod(d-π, 2π) - π`? Anyway this is equivalent to `wrap180(b - a)` but in radians, right? It could just be done that way too...

Reply:
Using `mod` appears to work fine. Updated in -> https://github.com/plotly/plotly.js/pull/2954/commits/939bdad1d24a2ca99bab9844d312ee65c514f7a7